docs/users.md

```diff
@@ -522,6 +600,66 @@ vcluster-0                                          2/2     Running   0
 
 The nginx pod is the one with the name `nginx-x-default-x-vcluster`.
 
+## Working with an external control plane
+
+In this section, we will show an example of creating an external control plane to adopt
+a kind cluster named `ext1`. This setup requires that the external cluster `ext1` 
+and the KubeFlex hosting cluster are on the same docker network.
+
+### Determining the endpoint for an external cluster using loopback address
+
+This is a common scenario when adopting kind or k3d. Ensure that both your 
+hosting and adopted clusters are on the same Docker network. For 
```

Comment:
"Ensure that both your hosting and adopted clusters are on the same Docker network" redundant with the remark just above that "This setup requires that the external cluster `ext1` and the KubeFlex hosting cluster are on the same docker network."

Reply:
removed the redundant sentence.